src/SystemTextJson.JsonDiffPatch/SystemTextJson.JsonDiffPatch.csproj

```diff
@@ -14,13 +14,13 @@
     <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net462'" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'== 'netStandard2.0' or '$(TargetFramework)'== 'net462' or '$(TargetFramework)'== 'net6.0'or '$(TargetFramework)'== 'net7.0'">
```

Comment:
I would like to remove net 6 & 7 from here but some people would consider that a breaking change.